### ISSUE TITLE
Code compilation error

### ISSUE THREAD

**Issue by pbhadauria2000** [swagger-api/swagger-codegen#4565]
##### Description

The latest code is giving compilation errors because of a recent code commit. 

_The declared package "io.swagger.codegen.python" does not match the expected package "io.swagger.codegen.bash"_ 

in file "swagger-codegen/modules/swagger-codegen/src/test/java/io/swagger/codegen/bash/BashTest.java"

##### Swagger-codegen version

Current master branch - 2.2.2-SNAPSHOT

##### Steps to reproduce

mvn package

##### Suggest a Fix

Renaming package "io.swagger.codegen.python" to "io.swagger.codegen.bash"

**Comment by wing328** [comment]
@pbhadauria2000 I couldn't repeat the issue in my MAC and Linux box.
```
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] swagger-codegen-project ............................ SUCCESS [  3.856 s]
[INFO] swagger-codegen (core library) ..................... SUCCESS [08:09 min]
[INFO] swagger-codegen (executable) ....................... SUCCESS [ 33.922 s]
[INFO] swagger-codegen (maven-plugin) ..................... SUCCESS [  5.079 s]
[INFO] swagger-generator .................................. SUCCESS [ 31.606 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 09:24 min
[INFO] Finished at: 2017-01-16T18:31:20+08:00
[INFO] Final Memory: 59M/724M
[INFO] ------------------------------------------------------------------------
```

but I agree the issue should be fixed and I'll file a PR shortly.

**Comment by pbhadauria2000** [comment]
Thanks @wing328

**Comment by wing328** [comment]
@pbhadauria2000 fixed via #4566. Please pull the latest master to try again.

**Comment by pbhadauria2000** [comment]
Works. Thanks @wing328
